robosuite/utils/opencv_renderer.py

```diff
@@ -46,5 +76,9 @@ def close(self):
         # NOTE: assume that @sim will get cleaned up outside the renderer - just delete the reference
         self.sim = None
 
+        if self.viewer is not None:
+            self.viewer.close()
+            self.viewer = None
```

Comment:
I didn't quite get this part. The implementation looks like mjviewer, but isn't this class designed for OpenCV? Can you double check or explain here what's reason changing this file?

Reply:
Since I couldn't use mjviewer for rendering then, I decided to use OpenCV. However, for real-time rendering, we need to use the function `update` to update robot state, which uses `mujoco.viewer`. When I was trying to transfer this logic from mjviewer to OpenCV, I need to init the viewer and close the viewer. That's why you can see many implementations that look like mjviewer in OpenCV. I was just for convenience. You can modify these files according to your needs. Hope the explanation clear for you!

Reply:
I agree with @zhuyifengzju that the changes in this file should not be included